src/keri/vdr/eventing.py

```diff
@@ -2007,12 +1994,8 @@ def processEscrows(self):
                                                 extype=kering.MissingRegistryError)
             self.reger.txnsb.processEscrowState(typ="credential-mae", processReply=self.processReplyCredentialTxnState,
                                                 extype=kering.MissingAnchorError)
-            self.reger.txnsb.processEscrowState(typ="credential-ooo", processReply=self.processReplyCredentialTxnState,
```

Comment:
If this change is right, maybe we should also remove the `credential-mre` escrow, not sure.